bin/cli.js

```diff
@@ -15,12 +15,14 @@ const argv = require("yargs")
                 alias: "R",
                 type: "string",
                 describe: "The mocha reporter to use, via tap-mocha-reporter",
-                default: "spec",
+                default: "jest",
```

Comment:
This is 🌶️, I realize -- if we want to keep the default as `spec` that's totally fine by me 😄

Reply:
I think `jest` reporter is fine, don't think people will complain